vanilla-installer/gui.py

```diff
@@ -22,185 +35,234 @@
 parser.add_argument("--safegui", type=bool)
 parser.add_argument("--litegui", type=bool)
 args = parser.parse_args()
-FONT = "Yu Gothic UI" if os.name == "nt" else "URW Gothic"
+FONT = "Inter"
 
 
 def run():
     """Runs the GUI."""
-    if args.litegui:
-        win = tkinter.Tk()
-    else:
-        win = tkinter.Tk(baseName="VanillaInstaller", className="VanillaInstaller")
-        # win.wm_attributes('-type', 'splash')
-
-    win.title(f'{"" if args.safegui else "🧰 " }Fabulously Optimized · VanillaInstaller')
-    win.config(bg=theme.load()["bg"])
-
-    if not args.safegui:
-        win.geometry("500x400")
-        win.minsize(500, 400)
-        # Disabled this so you can resize the window if you want
-        # win.maxsize(500, 400)
-
-    icon = pathlib.Path("media/icon.png").resolve()
-    try:
-        win.iconphoto(False, tkinter.PhotoImage(file=icon))
-    except tkinter.TclError:
-        logger.exception("Icon not found! Trying to download...")
-        download = requests.get(
-            "https://raw.githubusercontent.com/Fabulously-Optimized/vanilla-installer/main/media/icon.png"
+    app = QApplication([])
+    window = QMainWindow()
+    ui = Ui_MainWindow()
+    ui.setupUi(window)
+    get_versions_worker = Worker(ui.addVersions)
+    ui.threadpool.start(get_versions_worker)
+    window.show()
+    app.exec()
+class Ui_MainWindow(object):
+    def setupUi(self, MainWindow: QMainWindow):
+        if not MainWindow.objectName():
+            MainWindow.setObjectName("MainWindow")
```

Comment:
Seems at least on Windows, the window name is taken from here - so should probably be `VanillaInstaller` or something similar.

Reply:
can you try the new commit and see if the title changed?

Reply:
Once I'm home I'll try it yeah

Reply:
Yep, works now